graphql_server/aiohttp/graphqlview.py

```diff
@@ -75,6 +76,9 @@ def get_context(self, request):
     def get_middleware(self):
         return self.middleware
 
+    def get_validation_rules(self):
+        return self.validation_rules
```

Comment:
This should return the default validation rules if `validation_rules` is `None`:

`from graphql import specified_rules`

Reply:
okay! I'll make another commit!

Reply:
@jkimbo this is already handled at the core, though. Do you think that this is necessary?

https://github.com/graphql-python/graphql-core/blob/cc303671559822d9c3991e04fee170dce0f23553/src/graphql/validation/validate.py#L49-L50